okteto.yml

```diff
@@ -38,6 +38,31 @@ deploy:
     command: helm upgrade --install worker worker/chart --set image=${OKTETO_BUILD_WORKER_IMAGE}
   - name: Deploy API
     command: helm upgrade --install api api/chart --set image=${OKTETO_BUILD_API_IMAGE} --set load=${API_LOAD_DATA:-true}
+  - name: Reserve External IP
+    command: |
+      EXTERNAL_IP=""
+      while [ -z $EXTERNAL_IP ]; do
+        echo "Waiting for external ip..."
+        EXTERNAL_IP=$(kubectl get svc postgresql --template="{{range .status.loadBalancer.ingress}}{{.ip}}{{end}}")
+        [ -z "$EXTERNAL_IP" ] && sleep 10
+      done
+      echo "External IP ready: $EXTERNAL_IP"
+      echo "OKTETO_EXTERNAL_POSTGRES_ENDPOINTS_DB_URL=$EXTERNAL_IP" >> $OKTETO_ENV
+      envsubst < pd/README.md.tmpl > pd/README.md
```

Comment:
Here we wait for the external IP to be available, and then we pass the IP to OKTETO_ENV and to the readme so it's displayed both on the UI as well as on the PrivacyDynamic readme included in the development and preview environments.